### PR TITLE
chore: add agenda for June 16 meeting

### PR DESCRIPTION
**Pull request by bcoe**
Add UUID update to agenda.

### CODE REVIEW

File: 2020/06-16.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+## Agenda topic rules
+
+Please add topics you would like from the proposals from the [current charter](https://github.com/tc39/incubator-agendas/issues/5).
+
+- The champion deadline is 1:00 June 10, 2020.
+
+## Agenda items
+
+* An update on UUID ~ Benjamin Coe
```
Comment (syg) [inline 2020/06-16.md:16]: What we learned from the first incubator call was it'd be really helpful if champions could help scope the conversation by asking for feedback on specific items. Could you add one or two specific topics?

**Review by syg — APPROVED** [review]
lgtm, please add some specifics if you have them. Otherwise I'll merge tomorrow.

File: 2020/06-16.md (hunk 1 above)
Comment (bcoe) [reply 2020/06-16.md:16]: @syg I would be happy to flesh out the agenda a bit, is the request that I essentially rough out an agenda? If so, should we plan on the whole hour, or will there be a few topics of conversation?

Comment (syg) [reply 2020/06-16.md:16]: You got the floor for the whole hour. :)

Comment (syg) [reply 2020/06-16.md:16]: I've merged it for now for visibility. I'll add you as a contributor to edit the agenda directly.

**Pull request merged**